packaging/windows/BUILD.txt

```diff
@@ -223,7 +223,6 @@ B) Cross compile on Linux
 
     cmake -DCMAKE_TOOLCHAIN_FILE=../cmake/toolchain_mingw64.cmake \
         -DCMAKE_INSTALL_PREFIX=${INSTALL_PREFIX} \
-        -DRUNTIME_INSTALL_PREFIX=${RUNTIME_PREFIX} \
```

Comment:
Isn't this breaking cross-compilation?

Reply:
It isn't used anywhere. It is also not a CMAKE variable.